examples/minimal/src/layouts/Default.astro

```diff
@@ -0,0 +1,135 @@
+---
+import { Outlet } from 'astro/components';
+---
+
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<meta name="viewport" content="width=device-width" />
+		<meta name="generator" content={Astro.generator} />
+		<title>Astro</title>
+	</head>
+	<body>
+		<header>
+			<h1>My Website</h1>
+		</header>
+
+		<Outlet id="nav">
+			<slot name="nav" />
+		</Outlet>
```

Comment:
Main pattern for defining `Outlets`, wrap `Layout` slots in them.

`Outlet` requires a stable, unique `id`